Tests/test_content.py

```diff
@@ -23,7 +23,10 @@
 import requests
 import demisto_client.demisto_api
 from demisto_client.demisto_api.rest import ApiException
-from slackclient import SlackClient
+try:
```

Comment:
why is this needed?

Reply:
We have an old slack in our dockerimage.
On local envs mostly we will get the new slack.

Reply:
please comment why this is needed